README.md

```diff
@@ -155,7 +155,7 @@ In addition to the error handler that is defined in the container, the runner us
 A temporary error handler is needed to handle the creation of configuration and container instances,
 then the error handler configured in your application configuration will be used.
 
-By default, the temporary error handler uses plain text renderer and logging to a file. You can override this as follows:
+By default, the temporary error handler uses HTML renderer and logging to a file. You can override this as follows:
```

Comment:
What are the advantages of HTML renderer for logs?

Reply:
This is not for logs, but for rendering the error to the user.